Symfony/CS/Tests/Config/ConfigTest.php

```diff
@@ -23,7 +23,7 @@ public function testThatDefaultFinderWorksWithDirSetOnConfig()
         $config->setDir(__DIR__.'/../Fixtures/FinderDirectory');
 
         $iterator = $config->getFinder()->getIterator();
-        $this->assertSame(1, count($iterator));
+        $this->assertCount(1, $iterator);
```

Comment:
the issue with `assertCount` only effects HHVM and iterators, as some version(s) of PHPUnit tried to rewind the iterator when it is not possible.